awx/api/urls/urls.py

```diff
@@ -119,6 +120,7 @@
 v2_urls = [
     url(r'^$', ApiV2RootView.as_view(), name='api_v2_root_view'),
     url(r'^credential_types/', include(credential_type_urls)),
+    url(r'^credential_input_sources/', include(credential_input_source_urls)),
```

Comment:
If there a reason to have these at a global level, or is `/api/v2/credentials/N/input_sources/` good enough for our needs?